duckduckgo/iOS
# improve network detection and visualisation

Reviewer: Mia, Caine or Craig
Asana: https://app.asana.com/0/414235014887631/489057161850496
CC:

**Description**:

Some refactoring to separate the concerns of Disconnect Me Trackers from Detected Trackers, but mainly changes to show trackers without a network under a heading of their own, while still contributing individually to the network count.

Also added an icon for Yahoo, since it's considered a major tracker.

**Steps to test this PR**:

Test 1

1. Visit nytimes.com with protection on
1. A couple of trackers will be blocked and increment the count
1. Open the network tracker detail and they will both be under the "Unknown tracker network" header

Test 2

1. Turn off privacy protection
1. The number of trackers will increase dramatically
1. The number of networks will be around 15 but the trackers on the list will be higher
1. The Unknown tracker network group will be at the bottom of the view
1. Check the yahoo icon is visible

Test 3

1. Visit enough sites to generate a network leaderboard and check it looks OK (hopefully the yahoo icon will appear on the top 3 to confirm the icon)


###### Reviewer Checklist:


###### PR DRI Checklist:
